From 359a67898c4279f8feed653973f84c0e2f569517 Mon Sep 17 00:00:00 2001
From: David Greaves <david@dgreaves.com>
Date: Sun, 16 Jun 2019 18:17:43 +0100
Subject: [PATCH] Do not build with zstd. Set prefix to /usr

Signed-off-by: David Greaves <david.greaves@jolla.com>
---
 Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index e6d4609..08b131b 100644
--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,4 @@
-prefix=/usr/local
+prefix=/usr
 bindir=$(prefix)/bin
 libdir=$(prefix)/lib
 mandir=$(prefix)/man
@@ -8,9 +8,9 @@ zlibbundled=$(zlibdir)/libz.a
 zlibldflags=$(zlibbundled)
 zlibcppflags=-I$(zlibdir)
 pylibprefix=/
-CFLAGS = -fPIC -O2 -Wall -g -DWITH_ZSTD=1
+CFLAGS = -fPIC -O2 -Wall -g
 CPPFLAGS = -fPIC -DDELTARPM_64BIT -DBSDIFF_NO_SUF -DRPMDUMPHEADER=\"$(rpmdumpheader)\" $(zlibcppflags)
-LDLIBS = -lbz2 $(zlibldflags) -llzma -lzstd
+LDLIBS = -lbz2 $(zlibldflags) -llzma
 LDFLAGS =
 PYTHONS = python python3
 

base-commit: c5e0ca7482e2cfea5e4d902ffe488e0a71ed3e67
-- 
2.11.0

